### PR TITLE
[one-cmds] Add Fold DepthwiseConv2D optimization option

### PR DESCRIPTION
**Pull request by ai-moiseev**
This commit adds `fold_depthwise_conv_2d` optimization option to `one-build` config.

For #7505 

Signed-off-by: Alexander Moiseev <a.moiseev@samsung.com>

### CODE REVIEW

**Comment by binarman** [comment]
There are also `how-to-use-one-commands.txt` file nearby. Did you add this command there?

**Comment by seanshpark** [comment]
@ai-moiseev , @binarman , code itself is OK
But while looking at the name and comparing with other pass options,
- we used `Conv2D` as just `conv`
- `DepthwiseConv2D` as just `dwconv`

As to match with other options, (although existing may seem odd), what do you think about chanigng (including existing codes) option name to `fold_dwconv` ? I apologize not pointing this out earlier...

**Comment by ai-moiseev** [comment]
@seanshpark I've created renaming PR for circle2circle, PTAL #7617
It seems it doesn't break any tests... when it will be merged, I'll rebase this PR.

File: compiler/one-cmds/how-to-use-one-commands.txt [hunk 1]
```diff
@@ -152,6 +152,7 @@ Current transformation options are
 - disable_validation : This will turn off operator validations.
 - fold_add_v2 : This removes AddV2 operation which can be folded
 - fold_cast : This removes Cast operation which can be folded
+- fold_dwconv : This folds Depthwise Convolution operation which can be folded
```
Comment (seanshpark) [inline compiler/one-cmds/how-to-use-one-commands.txt:155]: (optional) maybe you may want to swap with below item?

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

File: compiler/one-cmds/how-to-use-one-commands.txt (hunk 1 above)
Comment (ai-moiseev) [reply compiler/one-cmds/how-to-use-one-commands.txt:155]: Sorry, I've read this comment a bit late... Here's PR which fixes sorting order: #7619 :)

**Pull request merged**